libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -478,6 +478,18 @@ void AP_AHRS::update_DCM()
     // }
 }
 
+#if AP_AHRS_SIM_ENABLED
```

Comment:
This is here because in future PRs this becomes an iteration across all of the backends, calling their update function and then their get_results function (and then, perhaps, updating the canonical AHRS results, which is what `copy_estimates_from_backend_estimates` does)

In far-future functions the `update()` might happen in a thread....